great_expectations/checkpoint/types/checkpoint_result.py

```diff
@@ -300,18 +306,21 @@ def to_json_dict(self) -> dict:
         due to the location of the "great_expectations/types/__init__.py" and "great_expectations/core/util.py" modules
         make this refactoring infeasible at the present time.
         """
-        serializeable_dict: dict = {
+        serializable_dict: dict = {
+            "validation_result_url": self.validation_result_url,
             "run_id": self.run_id.to_json_dict(),
             "run_results": convert_to_json_serializable(
                 data=recursively_convert_to_json_serializable(test_obj=self.run_results)
             ),
             "checkpoint_config": self.checkpoint_config.to_json_dict(),
             "success": convert_to_json_serializable(data=self.success),
         }
-        serializeable_dict = recursively_convert_to_json_serializable(
-            test_obj=serializeable_dict
+        if not self.validation_result_url:
+            serializable_dict.pop("validation_result_url")
+        serializable_dict = recursively_convert_to_json_serializable(
+            test_obj=serializable_dict
         )
-        return serializeable_dict
+        return serializable_dict
```

Comment:
I would avoid the extra assignment (`=`) operation.
```python
        if not self.validation_result_url:
            serializable_dict.pop("validation_result_url")
        return recursively_convert_to_json_serializable(serializable_dict)
```

Reply:
Although You shouldn't need to call this method at all. All the members are already json serializable.
```python
        if not self.validation_result_url:
            serializable_dict.pop("validation_result_url")
        return serializable_dict
```